client/src/components/PhotoGrid/PhotoGrid.jsx

```diff
@@ -20,9 +25,36 @@ function PhotoGrid(props) {
   };
 
   return (
-    <div>
-      {photos.slice(0, 5).map((photo) => <PhotoItem photo={photo} key={photo.id} />)}
-      <Button label="Show All Photos" />
+    <div className={classes.photogrid}>
+      {/* {photos.slice(0, 5).map((photo) => (
+        <PhotoItem
+          photo={photo}
+          setSelectedPhoto={setSelectedPhoto}
+          key={photo.id}
+        />
+      ))} */}
+      <div className={classes.left}>
+        {photosLoaded ? <PhotoItem photo={photos[0]} setSelectedPhoto={setSelectedPhoto} /> : null}
+      </div>
+      <div className={classes.right}>
+        <div className={classes.upperright}>
+          {photosLoaded ? <PhotoItem photo={photos[1]} setSelectedPhoto={setSelectedPhoto} /> : null}
+          {photosLoaded ? <PhotoItem photo={photos[3]} setSelectedPhoto={setSelectedPhoto} /> : null}
+        </div>
+        <div className={classes.lowerright}>
+          {photosLoaded ? <PhotoItem photo={photos[2]} setSelectedPhoto={setSelectedPhoto} /> : null}
+          {photosLoaded ? <PhotoItem photo={photos[4]} setSelectedPhoto={setSelectedPhoto} /> : null}
+        </div>
+      </div>
+      <Button style={{
```

Comment:
Just want to know why in line styling was used here? is it unique to just this component? can the styling be wrapped around a div and applied there?

Reply:
Styling has been moved into the Button component